py3status/core.py

```diff
@@ -596,9 +596,12 @@ def notify_user(self, msg, level='error', rate_limit=None, module_name=''):
         msg_hash = hash(u'{}#{}#{}'.format(module_name, limit_key, msg))
         if msg_hash in self.notified_messages:
             return
+        elif module_name:
+            x = 'Module `%s` sent a notification. "%s"' % (module_name, msg)
```

Comment:
`x` is a terrible name.  how about `log_msg` or something similar

Reply:
You drive a hard bargain. Done.